SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc

```diff
@@ -12,6 +12,147 @@ The content for this appendix may be pushed to SDPi 1.x.
+As pointed out above, requirements interoperability (RI) based on robust model-based metadata is a core, innovative aspect of this SDPi profiles specification.  Given the ultimate intent to realize this description as a _Model Centric (MC) single-source-of-truth, computable, simulatable, verifiable and validatable system of systems interoperability specification_, and recognizing that it will take a significant transition period from a document-centric approach to a model-centric approach, a simplified requirements model is provided below but is aligned with the <<omg_sysml_2-0_spec>> section 7.20 Requirements language.  Of course, that specification provides for significantly more detailed and complext modeling, the general constructs may be used in this document to start the transition toward that model.  Note that SysML 2.0 also better supports model interoperability (tool-independent model exchange) and _Model-based Systems Engineering (MBSE)_ (see https://en.wikipedia.org/wiki/Model-based_systems_engineering[MBSE Wikipedia article and references]), as well as the <<ihe_eu_experience_2021_presentation_cooper_schlichting>> for an overview presentation of MBSE, MedTech system V&V, and IHE Conformity Assessment.
+
+It should be further noted that though conformity testing aspects are beyond this SDPi specification, the modeling constructs used below will also be integrated with <<omg_sysml_2-0_spec>> section 7.23 Verification Cases, to provide for advanced V&V of interoperable system components and entire systems of products.
+
+==== SDPi Requirements Core Model
+
+To formally integrate requirements in to this specification, the following requirements model provides the starting point:
+
+.SDPi Requirements - Core Model
+
+image::../images/vol1-diagram-sdpi-req-types-model.svg[align=center]
+
+This model identifies a set of requirement "types" that are formalized in the specification.  Each type is a source of requirements that are explicitly identified and formalized with appropriate metadata.
+
+[%autowidth]
+[cols="^1,4,^1,^1"]
+|===
+|Model Element |Description |AsciiDoc Attribute |Further Specified
+
+| SDPi Requirement
+| A defined stakeholde-imposed constraints that must be satisfied for a design solution to be valid.  This is an {abstract} element.
+| sdpi_requirement
+| See subtypes
+
+| SDPi Requirement Group
+| Two or more SDPi Requirements may be collected into a group that is focused around a specific _subject_ area.
+| sdpi_requirement_group
+|
+
+| Usage
+| Requirement utilized in a specific use context that provides for its satisfaction.
+| sdpi_requirement_usage
+| 
+
+| Use Case Feature
+| A functional "feature" requirement based on clinical use case scenarios.
+| mdi_requirement_use_case
+| See TF-1 Appendix C, gherkin model
+
+| Ref. Standard ICS
+| Requirement definitions that are specified in a normative reference.
+| sdpi_requirement_ref_standard
+|
+
+| SES
+|
+| sdpi_requirement_ses
+| See SES section ... WHERE IS THAT?
```

Comment:
Add a todo that this table needs some work?

Reply:
This is true of the entire TF-1 Appendix A ... which is a work-in-progress, to be finished before integration into master.